src/index.js

```diff
@@ -46,13 +47,16 @@ window.patternfly = require('patternfly/dist/js/patternfly')
 window.selectpicker = require('bootstrap-select/js/bootstrap-select.js')
 window.combobox = require('patternfly-bootstrap-combobox/js/bootstrap-combobox.js')
 
-function renderApp (store: Object) {
+function renderApp (store: Object, errorThrower: Object) {
   ReactDOM.render(
-    <Provider store={store}>
-      <IntlProvider locale={locale} messages={getSelectedMessages()}>
-        <App history={store.history} />
-      </IntlProvider>
-    </Provider>,
+    <GlobalErrorBoundary>
+      <Provider store={store}>
+        <IntlProvider locale={locale} messages={getSelectedMessages()}>
+          <App history={store.history} errorThrower={errorThrower} />
```

Comment:
I see that you are bridging the saga error handler through to the react error stack with this.  I'm not 100% sold on the idea as it may end up trapping errors that may not be real errors -- I haven't read up on error handling in saga and redux in general.

But ... the react way is to create a new component for this instead of adding another purpose to an existing component.

So .... rename `errorThrower` to `sagaErrorBridge` and then ...
````jsx
<GlobalErrorBoundary>
  <React.Fragment>
    <RethrowSagaErrorAsReactError errorBridge={sagaErrorBridge} />
    <Provider store={store}>
    ......
    </Provider>
  </React.Fragment>
<GlobalErrorBoundary>
````

The `Fragment` may not be necessary based on how `GlobalErrorBoundary` handles its children.

`App` should remain untouched by any of the error handling code.

Create `RethrowSagaErrorAsReactError` as a component that has the bits you currently have in `App`.

`ErrorThrower` is too generic of a name.  Name is something specific because its job will be to:
  - collect components that will bridge errors into react (not forgetting to also release the component when the component is unmounted)
  - listen for error from saga
  - when an error occurs in saga, broadcast it out to all of its collect components